.gitattributes

```diff
@@ -1,6 +1,9 @@
-#
-# https://help.github.com/articles/dealing-with-line-endings/
-#
-# These are explicitly windows files and should use crlf
-*.bat           text eol=crlf
+# Documentation: https://www.git-scm.com/docs/gitattributes
 
+# Set default behavior to automatically normalize line endings.
+* text=auto eol=lf
+
+# Force batch scripts to always use CRLF line endings so that if a repo is accessed
```

Comment:
This comment seems overly specific, there are other reasons to use proper line endings for Windows files than WSL

Reply:
To be honest it was a copy/paste from someone else and I never questionned this comment so far. But yeah, let's fix that since anyway I find the comment just way too complicated to say "things may go wrong on Windows without that".

Reply:
I swear you just want me to rego through all the cloud repositories! You want me to die xD

Reply:
Fixed on all PRs! ^^

Reply:
> I swear you just want me to go again through all the cloud repositories! You want me to die xD

Sorry about that! But, you should probably have waited for this one to merge before opening PRs to the other repos to avoid such a situation.

Reply:
Yeah no worry I was purely joking xD
That's just I didn't expect to get such a careful review on such secondary file that no one tend to take care whereas it's still important x)
Furthermore, that's a file I already used a lot in several projects an noone provided me any feedback

So yes, I mainly found this story funny in the end, that's all xD
But you're absolutely right!

Thank you for the approval :)